app/build.gradle

```diff
@@ -42,6 +43,10 @@ android {
         }
     }
 
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
```

Comment:
As I remember starting from some gradle version kotlin is included to sources by default, can it be done used in project gradle version?

Reply:
Without this block, gradle doesn't process Kotlin files as sources. The gradle version doesn't change anything

Reply:
https://developer.android.com/kotlin/add-kotlin#source